.github/workflows/tests.yml

```diff
@@ -28,17 +28,27 @@ jobs:
       - name: Upgrade pip
         run: python -m pip install --upgrade pip
```

Comment:
I guess we don't need that "Upgrade pip" anymore?

Reply:
Done